app/views/facility_accounts_reconciliation/_table.html.haml

```diff
@@ -22,5 +21,4 @@
               .order-detail-extra.order-detail-note= order_detail.note
 
         %td= show_actual_total(order_detail)
-        %td= text_field_tag "order_detail[#{order_detail.id}][notes]", begin params[:order_detail][order_detail.id.to_s][:notes] rescue "" end
-  = render partial: "action_row"
+        %td= text_field_tag "order_detail[#{order_detail.id}][reconciled_note]", begin params[:order_detail][order_detail.id.to_s][:reconciled_note] rescue "" end
```

Comment:
Yeah, I originally just didn't want to try thinking through why this was there. It appears to be a funky way around doing `params.try(:[], :order_detail).try(:[], order_detail.id.to_s).try(:[], :reconciled_note)` or if were were on ruby 2.3 `params.dig(:order_detail,...)`.

It turns out the only place we're using a direct render is in the `index` view where these parameters won't be there, so we can simply remove the second `text_field_tag` argument.